packages/example-log-extension/src/component.ts

```diff
@@ -5,12 +5,11 @@
 
 import {EXAMPLE_LOG_BINDINGS} from './keys';
 import {Component, ProviderMap} from '@loopback/core';
-import {TimerProvider, LogActionProvider, LogLevelProvider} from './';
+import {TimerProvider, LogActionProvider} from './';
 
 export class LogComponent implements Component {
   providers?: ProviderMap = {
     [EXAMPLE_LOG_BINDINGS.TIMER]: TimerProvider,
     [EXAMPLE_LOG_BINDINGS.LOG_ACTION]: LogActionProvider,
-    [EXAMPLE_LOG_BINDINGS.APP_LOG_LEVEL]: LogLevelProvider,
```

Comment:
The purpose of having a `LogLevelProvider` was to provide a default `log` level when the component is bound. Should this not be the case anymore? If so, it should be updated in the readme.

Reply:
As I mentioned in the PR description, the log level should be set by the app. Having a provider for the default value is overkill. There are two perspectives:

1. The LogAction has a optional (to be added by #946) dependency of `level`. The default is `WARN`.
2. From the component's view, forcing it to contribute a provider for a single default value is not ideal. That's why I'm proposing to allow components to expose bindings of any flavor in #929.

I'll update the README.

Reply:
I like the idea of changing the log level into an optional dependency that has the default value configured here in the component, and getting rid of LogLevelProvider along the way 👍